cloudx/handler.go

```diff
@@ -480,14 +500,14 @@ func (h *SnakeCharmer) ListProjects() ([]cloud.Project, error) {
 	if err != nil {
 		return nil, err
 	}
-
```

Comment:
Can you please revert these newline changes? They make the code less readable. We try to separate blocks of logic using a singular newline :)